mi439.json

```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "vytska69 & orel6505",
+        "oem": "redmi",
+        "device": "xiaomi sdm439 devices",
+        "filename": "crDroidAndroid-12.1-20220413-mi439-v8.4.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/mi439/8.x/crDroidAndroid-12.1-20220413-mi439-v8.4.zip/download",
+        "timestamp": 1649853111,
+        "md5": "122538d3205b9f8ef3b14f56455bee27",
+        "sha256": "355d8d563584a8e09fe32614e0dfa640e07ee19373eeb11920a07886dfe177f0",
+        "size": 1038859044,
+        "version": "8.4",
+        "buildtype": "monthly",
+        "forum": "https://forum.xda-developers.com/t/12-arm64-mi439-crdroidandroid-v8-official.4431227/",
+        "gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-SL/",
+        "firmware": "",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "https://sourceforge.net/projects/rom-releases/files/pbrp-recovery/PBRP-mi439-3.1.0-20220410-1239-UNOFFICIAL.zip/download",
+        "paypal": "",
+        "telegram": "https://t.me/crdroid_pine",
+        "dt": "https://github.com/randomas3000/android_device_xiaomi_mi439",
```

Comment:
dt links need to point to crDroid organization on GitHub

Reply:
okay, i will do